test/Test/Migration.hs

```diff
@@ -144,14 +136,8 @@ migrationScriptV1 =
 emptyCode :: UContractRouter ver
 emptyCode = mkUContractRouter (Lorentz.drop # nil # pair)
 
-ownerAddress :: Address
-ownerAddress = genesisAddress3
-
 redeemAddress_ :: Address
-redeemAddress_ = ownerAddress
+redeemAddress_ = genesisAddress3
```

Comment:
Do I understand correctly that it does not matter which address do we pick here? Looks a bit suspicious now when we support both pure and on-chain runs.